leaflet-geoman.d.ts

```diff
@@ -716,12 +757,39 @@ declare module 'leaflet' {
             /** allow self intersections (default:true). */
             allowSelfIntersectionEdit?: boolean;
 
-            /** Disable the removal of markers/vertexes via right click. (default:false). */
+            /** Disable the removal of markers via right click / vertices via removeVertexOn. (default:false). */
             preventMarkerRemoval?: boolean;
 
             /** If true, vertex removal that cause a layer to fall below their minimum required vertices will remove the entire layer. If false, these vertices can't be removed. Minimum vertices are 2 for Lines and 3 for Polygons (default:true). */
             removeLayerBelowMinVertexCount?: boolean;
 
+            /** Leaflet layer event to add a vertex to a Line or Polygon, like dblclick. (default:click). */
+            addVertexOn?:
+                | 'click'
+                | 'dblclick'
+                | 'mousedown'
+                | 'mouseover'
+                | 'mouseout'
+                | 'contextmenu';
+
+            /** A function for validation if a vertex (of a Line / Polygon) is allowed to add. It passes a object with `[layer, marker, event}`. For example to check if the layer has a certain property or if the `Ctrl` key is pressed. (default:undefined). */
+            addVertexValidation?: undefined | VertexValidationHandler;
```

Comment:
is this working? undefined or the function

Reply:
These 2 scenarios are valid based on these changes:

```
const b: L.PM.DrawModeOptions = {
    addVertexValidation: undefined,
};

const c: L.PM.DrawModeOptions = {
    addVertexValidation: e => { return false; },
};
```

Reply:
Ok thanks